mypy/nodes.py

```diff
@@ -2360,6 +2363,18 @@ def type(self) -> 'Optional[mypy.types.Type]':
         else:
             return None
 
+    def copy(self) -> 'SymbolTableNode':
```

Comment:
The dataclasses backport only works on 3.6 since dataclasses' class based declaration syntax requires PEP 526. It will be a couple years before we can use it (3.5 goes out of support in late 2020).